changelog/1389.trivial.rst

```diff
@@ -0,0 +1 @@
+Refactored all test functions in :file:`test_validators.py` and :file:`test_checks.py` to be parameterized with `pytest.mark.parametrize`.
```

Comment:
```suggestion
Refactored all tests in :file:`test_validators.py` and :file:`test_checks.py` to be parameterized with `pytest.mark.parametrize`.
```

Just because tests are a combination of functions, classes, and methods.